.ci/Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+def getBuildConfig() {
+	if(env.BRANCH_NAME == "master") {
+		return "Release"
+	}
+	return "Debug"
+}
+
+pipeline {
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '10', artifactNumToKeepStr: '5'))
+    }
+    agent {
+	    node {
+            label 'nuget&&msbuild&&windows'
+		}
+    }
+	environment {
+		BUILD_CONFIG = getBuildConfig()
+		NUGET_CREDENTIALS = credentials("ef205e51-11e9-4ff0-b1a8-e9354fcc4eae")
+		NUGET_SOURCE = "https://api.bintray.com/nuget/xillio/nuget"
+	}
+    stages {
+        stage('Clean') {
+            steps {
+                 bat "nuget restore"
+                 bat "msbuild TusClient.proj /p:Config=${env.BUILD_CONFIG} /t:Clean"
+            }
+        }
+        stage('compile & build locally') {
+            steps {
+                 bat "msbuild TusClient.proj /p:Config=${env.BUILD_CONFIG} /t:Build"
+            }
+        }
+        stage('Publish') {
+            steps {
```

Comment:
Do we still want to publish every branch?

Reply:
No, i don't think it's a good idea

Reply:
There is no need for the double brackets. Also your formatting is kinda all over the place in this file.